prometheus_client/core.py

```diff
@@ -29,6 +29,11 @@
 _MINUS_INF = float("-inf")
 _INITIAL_MMAP_SIZE = 1 << 20
 
+pack_integer = struct.Struct(b'i').pack_into
```

Comment:
put an _ in front of these so they aren't exported

Reply:
ok